.prettierrc

```diff
@@ -0,0 +1,5 @@
+{
+  "singleQuote": true,
```

Comment:
Do we actually use Pretier?

Reply:
I personally use it for sure while development. It is up to the developer to enable it for the redactor/IDE.